project.clj

```diff
@@ -9,6 +9,9 @@
                  [clj-pdf "2.3.1"]
                  [clj-time "0.15.1"]
                  [cljs-ajax "0.8.0"]
+                 [cljsjs/react "16.8.3-0"]
```

Comment:
could add a comment here that these override the react inside reagent, and can be removed once we upgrade reagent